README.md

```diff
@@ -330,6 +330,30 @@ float remapTextureValue(
 
 Copyright © 2018, adidas AG, Released under the [Apache 2.0 License](LICENSE)
 
+## Use Cases
```

Comment:
- I think the use cases section should be placed after `1. Description`.
- Add this two sections to the index.